.gitignore

```diff
@@ -34,7 +34,7 @@ captures/
 
 # IntelliJ
 *.iml
-.idea/*
+.idea/
```

Comment:
Please check this once

Reply:
Have checked. Both have same effect. Exclude .idea and its contents from Git